### PR TITLE
handler to log the result path to a named logger

### PR DESCRIPTION
**Pull request by tylerroyal**
moved wadl2checker back to where it should have been

### CODE REVIEW

**Comment by RackerWilliams** [comment]
Crap.  I can't believe we missed the CLI tools being in the wrong place.  Can you also move wadl2checker and put that in a different pull request.  Would probably like to release just to fix.

Thanks

**Comment by RackerWilliams** [comment]
Actually got that backwards you move wadl2checker...but just deleted wadl2dot.  Please restore and move to the correct place.

**Comment by RackerWilliams** [comment]
retest this please

File: core/src/main/scala/com/rackspace/com/papi/components/checker/handler/ApiCoverageHandler.scala [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.rackspace.com.papi.components.checker.handler
+
```
Comment (RackerWilliams) [inline core/src/main/scala/com/rackspace/com/papi/components/checker/handler/ApiCoverageHandler.scala]: Missing license statement.

File: core/src/test/scala/com/rackspace/com/papi/components/checker/handler/ApiCoverageHandlerTest.scala [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+import com.rackspace.com.papi.components.checker.util.IdentityTransformPool
+import com.rackspace.com.papi.components.checker.{BaseValidatorSuite, TestConfig, Validator}
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.core.LoggerContext
+import org.apache.logging.log4j.test.appender.ListAppender
+import org.mockito.Mockito
+import org.mockito.Mockito._
+import org.scalatest.BeforeAndAfter
+import org.scalatest.mock.MockitoSugar
+
```
Comment (RackerWilliams) [inline core/src/test/scala/com/rackspace/com/papi/components/checker/handler/ApiCoverageHandlerTest.scala]: This test is not running because you're missing:

@RunWith(classOf[JUnitRunner])

File: core/src/test/scala/com/rackspace/com/papi/components/checker/handler/ApiCoverageHandlerTest.scala [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+class ApiCoverageHandlerTest extends BaseValidatorSuite with MockitoSugar with BeforeAndAfter {
+  val apiCoverageHandler = new ApiCoverageHandler
+  val checkerXml =
+    <checker xmlns="http://www.rackspace.com/repose/wadl/checker" xmlns:json="http://json-schema.org/schema#"
```
Comment (RackerWilliams) [inline core/src/test/scala/com/rackspace/com/papi/components/checker/handler/ApiCoverageHandlerTest.scala]: I don't think this test actually highlights the right edge case.

Take a look at this for an example of one that does:

https://github.com/rackerlabs/api-checker/blob/master/core/src/test/scala/com/rackspace/com/papi/components/checker/ValidatorWADLOptSuite.scala

**Comment by RackerWilliams** [comment]
Also, please rebase 😄

**Comment by tylerroyal** [comment]
New case found that breaks this.
